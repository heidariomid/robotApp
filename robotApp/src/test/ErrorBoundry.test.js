import React from 'react';
import {shallow} from 'enzyme';
import ErrorBoundry from '../components/ErrorBoundry';

describe('<ErrorBoundry />', () => {
	it('renders components', () => {
		const wrapper = shallow(<ErrorBoundry />);
		expect(wrapper.debug()).toMatchSnapshot();
	});
});
