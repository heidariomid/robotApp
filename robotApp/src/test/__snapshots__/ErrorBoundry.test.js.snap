// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundry /> renders components 1`] = `""`;
